etc/profile-m-z/newsboat.profile

```diff
@@ -6,6 +6,11 @@ include newsboat.local
 # Persistent global definitions
 include globals.local
 
+noblacklist ${HOME}/.config/newsbeuter
+noblacklist ${HOME}/.config/newsboat
+noblacklist ${HOME}/.local/share/newsbeuter
+noblacklist ${HOME}/.local/share/newsboat
```

Comment:
Needs `blacklist`s in disable-programs.inc.

Reply:
ok